ydb/core/tx/schemeshard/schemeshard_import.cpp

```diff
@@ -112,7 +112,7 @@ void TSchemeShard::FromXxportInfo(NKikimrImport::TImport& import, const TImportI
             for (ui32 itemIdx : xrange(importInfo->Items.size())) {
                 FillItemProgress(this, importInfo, itemIdx, *import.AddItemsProgress());
             }
-            import.SetProgress(importInfo->IsDone()
+            import.SetProgress(importInfo->IsDone())
```

Comment:
Actually, that parenthesis is unnecessary, because there is no such method `IsDone` in `TImportInfo` struct. You need to add it like this:

```
    bool IsDone() const {
        return State == EState::Done;
    }
 ```

Reply:
ok, thanks.

Curious though, what is [this](https://github.com/ydb-platform/ydb/blob/fb2c723ca8e9b4b49f492db5625af6b897327de9/ydb/core/tx/schemeshard/schemeshard_info_types.h#L2844) ? a completely different method definition as it take it takes a parameter ? a contract which forces you to have there ? or just unnecessary code ?

Reply:
Understood, thanks.

Reply:
Thank you.